# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1340743795 14400
#      Tue Jun 26 16:49:55 2012 -0400
# Branch stable
# Node ID f06a1602102418580feeb9fd436544a0cc89852a
# Parent  980e2d5c83f75208d64f150b657ed32a3a5a807a
doc: trivial doc fix

* oop.txi: remove stray \ from @ifnottex table, not @iftex table (I
guess this fix wasn't so trivial after all).

diff --git a/doc/interpreter/oop.txi b/doc/interpreter/oop.txi
--- a/doc/interpreter/oop.txi
+++ b/doc/interpreter/oop.txi
@@ -544,17 +544,17 @@ width 0.6pt \tabskip=0pt\cr
 & $a .\hat b$ && power (a, b) && Element-wise power operator&\cr
 & $a \hat b$ && mpower (a, b) && Matrix power operator&\cr
 & $a < b$ && lt (a, b) && Less than operator&\cr
 & $a <= b$ && le (a, b) && Less than or equal to operator&\cr
 & $a > b$ && gt (a, b) && Greater than operator&\cr
 & $a >= b$ && ge (a, b) && Greater than or equal to operator&\cr
 & $a == b$ && eq (a, b) && Equal to operator&\cr
 & $a != b$ && ne (a, b) && Not equal to operator&\cr
-& $a & b$ && and (a, b) && Logical and operator&\cr
+& $a \& b$ && and (a, b) && Logical and operator&\cr
 & $a | b$ && or (a, b) && Logical or operator&\cr
 & $! b$ && not (a) && Logical not operator&\cr
 & $a'$ && ctranspose (a) && Complex conjugate transpose operator &\cr
 & $a.'$ && transpose (a) && Transpose operator &\cr
 & $a : b$ && colon (a, b) && Two element range operator &\cr
 & $a : b : c$ && colon (a, b, c) && Three element range operator &\cr
 & $[a, b]$ && horzcat (a, b) && Horizontal concatenation operator &\cr
 & $[a; b]$ && vertcat (a, b) && Vertical concatenation operator &\cr
@@ -581,17 +581,17 @@ width 0.6pt \tabskip=0pt\cr
 @item @tab a .^ b @tab power (a, b) @tab Element-wise power operator @tab
 @item @tab a ^ b @tab mpower (a, b) @tab Matrix power operator @tab
 @item @tab a < b @tab lt (a, b) @tab Less than operator @tab
 @item @tab a <= b @tab le (a, b) @tab Less than or equal to operator @tab
 @item @tab a > b @tab gt (a, b) @tab Greater than operator @tab
 @item @tab a >= b @tab ge (a, b) @tab Greater than or equal to operator @tab
 @item @tab a == b @tab eq (a, b) @tab Equal to operator @tab
 @item @tab a != b @tab ne (a, b) @tab Not equal to operator @tab
-@item @tab a \& b @tab and (a, b) @tab Logical and operator @tab
+@item @tab a & b @tab and (a, b) @tab Logical and operator @tab
 @item @tab a | b @tab or (a, b) @tab Logical or operator @tab
 @item @tab ! b @tab not (a) @tab Logical not operator @tab
 @item @tab a' @tab ctranspose (a) @tab Complex conjugate transpose operator @tab
 @item @tab a.' @tab transpose (a) @tab Transpose operator @tab
 @item @tab a : b @tab colon (a, b) @tab Two element range operator @tab
 @item @tab a : b : c @tab colon (a, b, c) @tab Three element range operator @tab
 @item @tab [a, b] @tab horzcat (a, b) @tab Horizontal concatenation operator @tab
 @item @tab [a; b] @tab vertcat (a, b) @tab Vertical concatenation operator @tab
